### ISSUE TITLE
Создай .h & .cpp файлы для начала 

### ISSUE THREAD

**Issue by ProfBat018** [RafaelBlackwood/TestingSystem#1]
Будь осторожен с рнкурсивным объявлением класса

**Comment by RafaelBlackwood** [comment]
Да-да, я когда загружал этот проект, я подумал об этом, я в будущем собираюсь его изменить и исправить все ошибки, ибо это мой старенький проект, нужно будет в ближайшее свободное время  этим заняться
